bciers/apps/registration/app/components/operations/registration/OperationInformationPage.tsx

```diff
@@ -42,25 +42,31 @@ export const createOperationInformationSchema = (
   );
 
   const oneOfOptions = purposes.map((purpose) => {
+    const isRegulatedProducts =
```

Comment:
Maybe `needsRegulatedProducts` or `hasRegulatedProducts`?

Reply:
I don't feel strongly at all however I think prefixing boolean variables with `is` is a common practice we have been using throughout our codebase. This keeps it in line with most of our other conditional variables! 

That said I don't mind changing it if you think otherwise.